CRITICAL - 2022-08-29 13:37:56 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `lrfoims...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\ProductManagement\Models\MenuModel.php(28): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(130): Modules\ProductManagement\Models\MenuModel->getDetails(Array)
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->menu()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-08-29 13:37:56 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:38:04 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `lrfoims...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\ProductManagement\Models\MenuModel.php(28): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(130): Modules\ProductManagement\Models\MenuModel->getDetails(Array)
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->menu()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-08-29 13:38:04 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:38:33 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `frbs_ro...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\RolesPermissionsModel.php(41): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\BaseController.php(66): Modules\UserManagement\Models\RolesPermissionsModel->getSecurityPermissions(Array)
#6 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(21): App\Controllers\BaseController->hasPermissionRedirect('orders')
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->index()
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-08-29 13:38:33 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:38:33 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `frbs_ro...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\RolesPermissionsModel.php(41): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\BaseController.php(66): Modules\UserManagement\Models\RolesPermissionsModel->getSecurityPermissions(Array)
#6 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(21): App\Controllers\BaseController->hasPermissionRedirect('orders')
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->index()
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-08-29 13:38:33 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:38:47 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `frbs_ro...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\RolesPermissionsModel.php(41): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\BaseController.php(66): Modules\UserManagement\Models\RolesPermissionsModel->getSecurityPermissions(Array)
#6 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(21): App\Controllers\BaseController->hasPermissionRedirect('orders')
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->index()
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-08-29 13:38:47 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:40:00 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `frbs_ro...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\RolesPermissionsModel.php(41): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\BaseController.php(66): Modules\UserManagement\Models\RolesPermissionsModel->getSecurityPermissions(Array)
#6 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(21): App\Controllers\BaseController->hasPermissionRedirect('orders')
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->index()
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-08-29 13:40:50 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `frbs_ro...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\RolesPermissionsModel.php(41): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\BaseController.php(66): Modules\UserManagement\Models\RolesPermissionsModel->getSecurityPermissions(Array)
#6 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(21): App\Controllers\BaseController->hasPermissionRedirect('orders')
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->index()
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-08-29 13:40:50 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:40:53 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT `lrfoims...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\ProductManagement\Models\MenuModel.php(28): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(130): Modules\ProductManagement\Models\MenuModel->getDetails(Array)
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->menu()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-08-29 13:40:53 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(Modules\OrderManagement\Controllers\Order), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:40:54 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT count(id...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lrfoims\modules\UserManagement\Models\UsersModel.php(60): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lrfoims\app\Controllers\Dashboard.php(14): Modules\UserManagement\Models\UsersModel->getUserData()
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-08-29 13:40:54 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Dashboard), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:40:57 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(247): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(624): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\lrfoims\app\Validation\UserRules.php(10): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Validation\Validation.php(306): App\Validation\UserRules->validateUser('superadmin', 'username,passwo...', Array, NULL)
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Validation\Validation.php(164): CodeIgniter\Validation\Validation->processRules('password', 'password', 'superadmin', Array, Array)
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Controller.php(181): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\lrfoims\app\Controllers\Security.php(33): CodeIgniter\Controller->validate(Array, Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): App\Controllers\Security->index()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Security))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 13:40:57 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Security), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 13:43:09 --> Unable to connect to the database.
#0 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Model.php(247): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\BaseModel.php(624): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\lrfoims\app\Validation\UserRules.php(10): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Validation\Validation.php(306): App\Validation\UserRules->validateUser('superadmin', 'username,passwo...', Array, NULL)
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Validation\Validation.php(164): CodeIgniter\Validation\Validation->processRules('password', 'password', 'superadmin', Array, Array)
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Controller.php(181): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\lrfoims\app\Controllers\Security.php(33): CodeIgniter\Controller->validate(Array, Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): App\Controllers\Security->index()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Security))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 13:43:09 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\lrfoims\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Security), true)
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(281): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-08-29 14:59:33 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(168): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 168, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->addOrderToCartInMenuList()
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 15:00:08 --> Undefined variable: availableOrderNumbers
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Views\menuList\index.php(43): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 43, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Modules\\OrderMa...', Array, true)
#4 C:\xampp\htdocs\lrfoims\app\Views\templates\index.php(9): view('Modules\\OrderMa...')
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('templates/index', Array, true)
#8 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(185): view('templates/index', Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->addOrderToCartInMenuList()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 15:01:05 --> Undefined variable: availableOrderNumbers
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Views\menuList\index.php(43): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 43, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Modules\\OrderMa...', Array, true)
#4 C:\xampp\htdocs\lrfoims\app\Views\templates\index.php(9): view('Modules\\OrderMa...')
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('templates/index', Array, true)
#8 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(185): view('templates/index', Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->addOrderToCartInMenuList()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 16:10:08 --> Undefined variable: carts
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Views\menuList\index.php(136): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 136, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Modules\\OrderMa...', Array, true)
#4 C:\xampp\htdocs\lrfoims\app\Views\templates\index.php(9): view('Modules\\OrderMa...')
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('templates/index', Array, true)
#8 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(137): view('templates/index', Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->menu()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 16:42:33 --> Undefined index: count_unavailable_admin_carts
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Views\menuList\index.php(42): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 42, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Modules\\OrderMa...', Array, true)
#4 C:\xampp\htdocs\lrfoims\app\Views\templates\index.php(9): view('Modules\\OrderMa...')
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('templates/index', Array, true)
#8 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(137): view('templates/index', Array)
#9 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->menu()
#10 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#11 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-08-29 17:08:30 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:08:41 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:12:45 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:13:40 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:15:27 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:15:39 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:20:46 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:22:45 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:22:51 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('9', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:23:08 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('3', '4')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:24:50 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:31:27 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:33:46 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:35:53 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-08-29 17:41:05 --> Undefined offset: 0
#0 C:\xampp\htdocs\lrfoims\modules\OrderManagement\Controllers\Order.php(83): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 83, Array)
#1 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(935): Modules\OrderManagement\Controllers\Order->placeOrder('2', '2')
#2 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(Modules\OrderManagement\Controllers\Order))
#3 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\lrfoims\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\lrfoims\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
